packages/mobile/src/web3/saga.ts

```diff
@@ -434,14 +451,9 @@ export function* switchToGethFromZeroSync() {
     // Note that this must happen after the sync mode is switched
     // as the web3.personal where the key is stored is not available in zeroSync mode
     yield call(ensureAccountInWeb3Keystore)
-
-    if (gethAlreadyStartedThisSession) {
-      // If geth is started twice within the same session,
-      // there is an issue where it cannot find deployed contracts.
-      // Restarting the app fixes this issue.
-      restartApp()
-    }
   } catch (e) {
+    const zeroSync3 = yield select(zeroSyncSelector)
+    Logger.info(TAG, `switchToGethFromZeroSync, zeroSync3 mode: ${zeroSync3}`)
```

Comment:
Looks like log line should be 
```js
Logger.info(TAG + '@switchToGethFromZeroSync', `zeroSync3 mode: ${zeroSync3}`)
```
to be consistent with the other statements.

Reply:
Good catch Jean, thanks! I'll fix to follow the `Logger.info(TAG, 'functionName', 'Log message')` format